animation_nodes/data_structures/attributes/attribute.pxd

```diff
@@ -0,0 +1,14 @@
+cdef enum AttributeType:
```

Comment:
Declare this as `cpdef` so that you can access them in Python. They should be accessible in Python in a manner similar to [PEP 435](https://www.python.org/dev/peps/pep-0435/).

Reply:
Thanks for the information. It is done.